src/azul/service/hca_response_v5.py

```diff
@@ -470,6 +473,17 @@ def choose_entry(_term):
             term_object_params = {'term': choose_entry(term), 'count': term['doc_count']}
             if 'myProjectIds' in term:
                 term_object_params['projectId'] = [bucket['key'] for bucket in term['myProjectIds']['buckets']]
+            try:
+                organism_age = contents['myTerms']['meta']['path'][-1] == 'organism_age'
+            except (ValueError, KeyError):
+                pass  # Not an organims_age term
+            else:
+                if organism_age:
+                    try:
+                        value, unit = term_object_params.get('term').split()
+                    except AttributeError:
+                        value, unit = [None, None]
+                    term_object_params['term'] = dict(value=value, unit=unit)
```

Comment:
```suggestion
            if contents['myTerms']['meta']['path'][-1] == 'organism_age':
                try:
                    value, unit = term_object_params['term'].split()
                except AttributeError:
                    value, unit = [None, None]
                term_object_params['term'] = dict(value=value, unit=unit)
```

`contents['myTerms']['meta']['path']` should always exist, added by `_annotate_aggs_for_translation()` to all aggs. Currently however this will break 2 test cases `test_file_search_response_add_facets()` and `test_projects_file_search_response()` because they use a hardcoded `facets_populated` value that did not go through the annotation.

Adding `'meta': {'path': ['contents', 'samples', 'organ']},` and `"meta": {"path": ["contents", "samples", "disease"]},` to the `myTerms` sections of `facets_populated` should fix that.